"npm install" command breaks installation when previously installed

I've installed the nodejs-driver on Windows 10 (and Windows Server 2012) using the npm command "npm install teradata-nodejs-driver". If I later run the command "npm install" the node_modules\teradata-nodejs-driver\node_modules\ directory is deleted, but everything else in the node_modules\teradata-nodejs-driver\ directory remains. It looks like the same happens when running "npm ci" or "npm update". Re-running the original command "npm install teradata-nodejs-driver" fixes the issue, as does uninstalling and re-installing the package. Without the directory mentioned above, this error is received when creating a teradata connection object:
```
Error: Cannot find module '@teradataprebuilt/fastcall-win32-node-v64'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    ...
```

Also, potentially related, is that the install command won't run if I use optional parameters. For example "npm install teradata-nodejs-driver --loglevel verbose" fails due to "--loglevel verbose" being passed as optional parameters to node_modules\teradata-nodejs-driver\scripts\install-dependencies.js.

Same. 

```
internal/modules/cjs/loader.js:589
    throw err;
    ^

Error: Cannot find module '@teradataprebuilt/fastcall-linux-node-v67'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
    at Function.Module._load (internal/modules/cjs/loader.js:513:25)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    ...
```

Fedora 30
Node 11.0.0
NPM 6.4.1

--- a/scripts/install-dependencies.js
+++ b/scripts/install-dependencies.js
@@ -1,6 +1,7 @@
 'use strict';
 
 var spawn = require('child_process').spawnSync;
+var path = require('path');
 
 function getSpawn(command, args, options) {
   if (!/^win/.test(process.platform)) { // linux
@@ -20,5 +21,7 @@ var abiVersion = process.versions.modules;
 var os = process.platform;
 var teradataPrebuiltFastcallVersion = '0.2.6';
 var teradataNativelibVersion = '1.0.0-beta.1';
+var app_root_dir = path.resolve(__dirname).split('node_modules')[0];
 
-getSpawn('npm', ['install', '--no-save', '@teradataprebuilt/fastcall-' + os + '-' + platform + '-v' + abiVersion + '@' + teradataPrebuiltFastcallVersion, '@teradataprebuilt/nativelib-' + os + '@' + teradataNativelibVersion]);
+process.chdir(app_root_dir);
+getSpawn('npm', ['install', '--save', '@teradataprebuilt/fastcall-' + os + '-' + platform + '-v' + abiVersion + '@' + teradataPrebuiltFastcallVersion, '@teradataprebuilt/nativelib-' + os + '@' + teradataNativelibVersion]);

--- a/src/teradata-connection.ts
+++ b/src/teradata-connection.ts
@@ -72,11 +72,11 @@ export class TeradataConnection {
     // Setup libpath for fastcall.library
     let libpath: string = '';
     if (process.platform === 'win32') {
-      libpath = __dirname + '/node_modules/@teradataprebuilt/nativelib-win32/teradatasql.dll';
+      libpath = 'node_modules/@teradataprebuilt/nativelib-win32/teradatasql.dll';
     } else if (process.platform === 'darwin') {
-      libpath = __dirname + '/node_modules/@teradataprebuilt/nativelib-darwin/teradatasql.dylib';
+      libpath = 'node_modules/@teradataprebuilt/nativelib-darwin/teradatasql.dylib';
     } else if (process.platform === 'freebsd' || process.platform === 'linux') {
-      libpath = __dirname + '/node_modules/@teradataprebuilt/nativelib-linux/teradatasql.so';
+      libpath = 'node_modules/@teradataprebuilt/nativelib-linux/teradatasql.so';
     }
 
     // Load native library with libpath and options

